src/Digdir.Domain.Dialogporten.WebApi/Digdir.Domain.Dialogporten.WebApi.csproj

```diff
@@ -8,13 +8,21 @@
 
     <ItemGroup>
         <PackageReference Include="Azure.Identity" Version="1.13.0" />
+        <PackageReference Include="Azure.Monitor.OpenTelemetry.AspNetCore" Version="1.2.0" />
         <PackageReference Include="FastEndpoints.Swagger" Version="5.29.0"/>
         <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0"/>
         <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.10" />
         <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="8.0.10" />
         <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.10" />
         <PackageReference Include="Microsoft.Azure.AppConfiguration.AspNetCore" Version="8.0.0" />
         <PackageReference Include="Serilog.AspNetCore" Version="8.0.3" />
+        <PackageReference Include="Npgsql.OpenTelemetry" Version="8.0.4" />
+        <PackageReference Include="OpenTelemetry" Version="1.9.0" />
+        <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.9.0" />
+        <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.9.0" />
+        <PackageReference Include="OpenTelemetry.Instrumentation.Http" Version="1.9.0" />
+        <PackageReference Include="OpenTelemetry.Instrumentation.Runtime" Version="1.9.0" />
+        <PackageReference Include="OpenTelemetry.Instrumentation.StackExchangeRedis" Version="1.9.0-beta.1" />
```

Comment:
Beta?

Reply:
Ja, det er kun RC og betaversjoner tilgjengelig for redis i OpenTelemetry. Vi kan evt. vente med å instrumentere Redis til det er mer modent?

Reply:
stemmer for å kjøre beta da ja 🤔 Er det noe ETA på å få det inn i fullversjon?

Reply:
Diskutert med Are, kutter ut Redis i denne PRen, kommer tilbake til det senere